github-for-jira.sd.yml

```diff
@@ -486,7 +528,37 @@ environmentOverrides:
             - github.stg.atlassian.com
           ip_whitelist:
             - public
-          routes: *blackholeSpammingIPs
+          upstream_address:
+            - address: '_Ref api-gateway.gateway.Host'
+              region: us-east-1 # The region that your endpoint is configured
+          routes: *globalEdgeRoutes
+
+      - type: api-gateway
+        name: gateway
+        attributes:
+          swaggerConfig:
+            configFile: "github-for-jira/api-gateway-${BITBUCKET_BUILD_NUMBER}.yml"
+            configParams:
+              AccountNumber: 752631980301
+              WebhooksQueueUrl: '_Ref sqs.webhooks.queueurl'
+              WebhooksQueueName: '_Ref sqs.webhooks.queuename'
+              WebhooksQueueRegion: '_Ref sqs.webhooks.queueregion'
+              ServerHost: 'github-for-jira.stg.services.atlassian.com'
+          stageName: live
+          dataType:
+            - UGC/Label                # name of GitHub org / Jira site
+            - PII/IndirectConfidential # name of GitHub org
+            - UGC/Configuration        # data about the installation of the GitHub app into Jira sites
+            - Security/Secret          # shared Connect secret
+            - UGC/PrimaryIdentifier    # references to GitHub entities (commits, pull requests, etc.) and Jira issues
+            - UGC/Primary
+
+      - name: rds
```

Comment:
I think last time I check it, it will raise some credit error after we deploy (or during deploy).
But worth solve it anyway.

Reply:
credit error?